internal/common/regions.go

```diff
@@ -23,11 +23,11 @@ func GetAWSRegions(ctx context.Context, cfg aws.Config, selectedRegions []string
 
 	log.Info("Starting GetAWSRegions function")
 
-	regionsToCheck := getRegionsToCheck(selectedRegions, log)
+	regionsToCheck := GetRegionsToCheck(selectedRegions, log)
 	return checkRegions(ctx, cfg, regionsToCheck, log)
 }
 
-func getRegionsToCheck(selectedRegions []string, log svc1log.Logger) []string {
+func GetRegionsToCheck(selectedRegions []string, log svc1log.Logger) []string {
```

Comment:
Don't pass in the logger here. I'm sorry I missed this before. Just pass in the Context